src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -57,18 +57,17 @@ setup_teuthology() {
     cd $TEMP_DIR
 
     cat <<EOF > t_reqs.txt
-apache-libcloud==2.2.1
+apache-libcloud==2.6.1
```

Comment:
Shouldn't we do this in https://github.com/ceph/teuthology/blob/master/requirements.txt / setup.py ? Versions there are completely different from these ones?

I removed `mysqlclient` dependency there so that we could install this directly from `pip install git+https://github.com/ceph/teuthology`. We could also ask @zmc to refresh https://pypi.org/project/teuthology/, so we can do a simpler `pip install teuthology==vX.Y.Z`

Reply:
i like your idea, @epuertat . but i think this is a subset of https://github.com/ceph/teuthology/blob/master/requirements.txt, which is just enough to launch teuthology locally. shall we install teuthology in a wholesale?

Reply:
If we don't need everything there (openstack packages?), we could pull out some deps to extras, same as I did for the coverage (https://github.com/ceph/teuthology/pull/1303/files). As is right now, time taken is double, but once created the virtualenv and compared to the test run length... Don't know, just for avoiding the mess of updating things in 2 different places?
```
35.410s total: pip install git+https://github.com/ceph/teuthology
14.671s total: pip install -r t_reqs.txt && git clone --depth 1 https://github.com/ceph/teuthology.git
```

Reply:
i added an flavor named "orchestra" in https://github.com/ceph/teuthology/pull/1367

Reply:
i will update this PR once the teuthology one gets merged.

Reply:
Thanks a lot, @tchaikov !

Reply:
@epuertat updated and repushed.